db/migrate/20240322132501_create_submission_error_tag.rb

```diff
@@ -0,0 +1,10 @@
+class CreateSubmissionErrorTag < ActiveRecord::Migration[7.1]
+  def up
+    Tenant.find_each do |tenant|
+      tenant.tags.find_or_create_by!(
+        name: 'Chyba odoslania',
```

Comment:
`Problematické správy` ? `Chybné správy`? Rozmyslam co vsetko pod to strcime a co by som cakal ze bude napisane vlavo v menu. "Chyba odoslania" mi znie divne.

Reply:
Chybne spravy napr. mi znie trochu matuco, kedze moze ist iba o vypadok UPVS alebo GB API. Problematicke spravy trochu lepsie, ale stale si pod tym nepredstavim, ze to moze byt iba docasna zalezitost.

Reply:
Co takto rovno "Problémové" = vacsinou su to pridavne mena (archivovane, vybavene, na prevzatie)

Reply:
hej, to asi sedi, zmenila som